src/main/ansible/vagrant.yml

```diff
@@ -18,55 +18,17 @@
   become: yes
   vars:
     local_test_vm_base_backend_port: 20150
+    dans_config_base: "/etc/opt/dans.knaw.nl"
   roles:
     - dans.local-test-vm-base
     - dans.local-yum-repo
+    - easy-auth-info
+    - easy-bag-store
 
-########### bag store
-
-- hosts: "test"
-  vars:
-    dans_config_base: "/etc/opt/dans.knaw.nl"
-    stores_dir: "/data/bag-stores"
-    bag_stores:
-      - { name: "pdbs",      base_dir: "{{ stores_dir }}/pdbs" }
-      - { name: "easy",      base_dir: "{{ stores_dir }}/easy" }
-      - { name: "pan",       base_dir: "{{ stores_dir }}/pan" }
-  become: yes
-  tasks:
-
-  - name: Installing bag-store package
-    yum:
-      name: "dans.knaw.nl-easy-bag-store"
-      state: latest
-
-  - name: Configure bag stores
-    lineinfile:
-        dest: "{{ dans_config_base }}/easy-bag-store/stores.properties"
-        line: "{{ item.name }}={{ item.base_dir }}"
-    with_items: "{{ bag_stores }}"
-
-  - name: Create empty bag-store base directories
-    file:
-      path: "{{ stores_dir }}/{{ item.name }}"
-      state: "directory"
-      owner: "easy-bag-store"
-      group: "easy-bag-store"
-      mode: "0775"
-    with_items: "{{ bag_stores }}"
-
-  - name: Create link to test data
-    file:
-      src: /vagrant/src/test/resources/vault/stores/pdbs/bags
-      dest: /home/vagrant/test-data
-      state: link
-
-  - service:
-      name: "easy-bag-store"
-      state: restarted
-      enabled: yes
+########### the solr core
 
-########### solr
+# TODO seems to need JAVA_HOME apparently set by other easy modules
+# TODO include_role (deprecated anyway) seems a problem within a role, but it shares vars with the other solr tasks
 
 - hosts: "test"
```

Comment:
I assume this thing is for installing Solr? Make a separate role for it as well...

Reply:
please read the second coemmt line: I did not manage to do so.

Reply:
I don't really understand. Can we look into this some time?